app/src/components/user/DeleteConfirmationModal/index.js

```diff
@@ -17,6 +18,8 @@ const DeleteConfirmationModal = ({
   const user = useSelector(getUserAuthDetails);
   const isWorkspaceMode = useSelector(getIsWorkspaceMode);
 
+  const enableTrash = useFeatureIsOn("enable-trash");
+
   const tryMoveToTrash = useCallback(() => {
```

Comment:
though the feature flag is triggered after the call to `handleRecordsDeletion`, it would be better to add a check here too. _since the name of the function co-relates to trash logic_